website/www/site/content/en/contribute/release-guide.md

```diff
@@ -584,7 +584,7 @@ See the source of the script for more details, or to run commands manually in ca
       1. Select repository `orgapachebeam-NNNN`.
       1. Click the Close button.
       1. When prompted for a description, enter “Apache Beam, version X, release candidate Y”.
-      1. Review all staged artifacts on https://repository.apache.org/content/repositories/orgapachebeam-NNNN/.
+      1. Review all staged artifacts in the `orgapachebeam-NNNN` repository on https://repository.apache.org/content/repositories.
```

Comment:
this wording was intentional. Each release, a repository is assigned a unique ID, which keeps changing. For example, right now there is https://repository.apache.org/content/repositories/orgapachebeam-1256. Perhaps we can make escape this link so that the broken-links parser doesn't pick it up?

Reply:
That's a good point. I think formatting this as text rather than making it a link is the right approach. I'll make that update.